userguide/sharing.rst

```diff
@@ -1499,8 +1499,12 @@ To configure iSCSI, click :guilabel:`WIZARD` and follow each step:
      This is required when the :guilabel:`Discovery Auth Method` is set
      to *CHAP* or *Mutual CHAP*.
 
-   * :guilabel:`IP`: Select the IP address associated with an
-     interface or the wildcard address of *0.0.0.0* (any interface).
+   * :guilabel:`IP`: Select the IP addresses associated with an
```

Comment:
The iSCSI Portals page doesn't even show network interfaces, so that reference is not needed.
Other conditions are essentially error handling, and don't need to be mentioned.
```
Select IP addresses to be listened on by the portal. Click :guilabel:`ADD
to add IP addresses with a different network port. The address
:literal:`0.0.0.0` can be entered to listen on all IPv4 addresses, or
:literal:`::` to listen on all IPv6 addresses.
```